types/src/generated/narwhal.rs

```diff
@@ -51,7 +58,7 @@ pub struct ValidatorData {
     #[prost(int64, tag="2")]
     pub stake_weight: i64,
     #[prost(message, optional, tag="3")]
-    pub address: ::core::option::Option<MultiAddr>,
+    pub primary_addresses: ::core::option::Option<PrimaryAddresses>,
```

Comment:
Could you also please update the `docs` and `protos` files in the [examples](https://github.com/MystenLabs/narwhal/tree/main/examples) as well?